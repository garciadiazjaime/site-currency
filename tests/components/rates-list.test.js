import React from 'react'
import { shallow } from 'enzyme'

import RatesList from '../../components/rates-list'

describe('Rate', () => {
  describe('main', () => {
    const baseRate = 'MXN'
    const rates = [{
      currency: 'currency'
    }, {
      currency: 'MXN'
    }, {
      currency: 'currency'
    }]
    const wrapper = shallow(<RatesList rates={rates} baseRate={baseRate} />)

    it('matches snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })

    it('renders all rates except baseRate', () => {
      expect(wrapper.find('Rate').length).toBe(2)
    })
  })
})
