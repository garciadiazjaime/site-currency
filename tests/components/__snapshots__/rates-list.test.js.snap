// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate main matches snapshot 1`] = `
<section
  id="rates-list"
>
  <Rate
    currency="currency"
    index={0}
    key="currency"
  />
  <Rate
    currency="currency"
    index={1}
    key="currency"
  />
</section>
`;
